content/tutorials/noetic_quickstart.md

```diff
@@ -0,0 +1,140 @@
+
+In the same terminal, build the stack 
+{{< highlight bash >}} $ source .bashrc && cd catkin_ws && catkin build {{< / highlight >}}
+
+## Downloading Foxglove Studio
+Since we can run the MuSHR stack now, we can use Foxglove Studio to visualize our robot and map. 
+
+First, download [Foxglove Studio](https://foxglove.dev/download). Foxglove is 
+still in development and has many features that are added frequently, so make sure to download the most
+recent version, or update it if you already have it downloaded.
+
+Open Foxglove Studio, and click the "Layouts" button on the left panel (second from top) and then click
+`Import Layout` button pictured below.
+{{< figure src="/tutorials/noetic_quickstart/foxglove_layout.png" width="400" >}} 
+
+Import the preset layout at:
+{{< highlight bash >}} mushr/mushr_utils/foxglove/foxglove_layout.json {{< / highlight >}}    
+
+Three panels should appear after selecting this layout, as pictured below. The left panel is for the datasource, the central panel is for the map, and the right panel is for teleop driving.
+
+{{< figure src="/tutorials/noetic_quickstart/layout_example.png" width="700" >}}                      
+
+The layout can be edited with the `Add Panel` button on the left sidebar if desired.
+
+## Connecting to Data With Foxglove Studio
+Now that Foxglove is set up, we can connect the visualization to our docker container ROS setup. Start the docker container:
+{{< highlight bash >}} $ mushr_noetic {{< / highlight >}}
+Once the `root` prompt appears, source the `.bashrc`. Note that this does not occur automatically in the docker container.
+
+{{< highlight bash >}} $ source .bashrc {{< / highlight >}}
+
+In the same terminal, start up the simulator with the command:
+
+{{< highlight bash >}} $ roslaunch mushr_sim teleop.launch {{< / highlight >}}
+
+After starting up, the simulator should print out a line similar to 
+
+{{< highlight bash >}} Rosbridge WebSocket server started at ws://0.0.0.0:9090 {{< / highlight >}}
+
+In foxglove, click the top button in the sidebar, labeled `Data Source`. Then select the Plus button in the left panel. This should open up an interface to connect to data. 
+
+{{< figure src="/tutorials/noetic_quickstart/data_panel.png" width="400" >}}
+
+Click the `Open Connection` button. Select `Rosbridge (ROS 1 & ROS 2)` as shown below.
+
+{{< figure src="/tutorials/noetic_quickstart/rosbridge.png" width="700" >}}
+
+Fill out the WebSocket URL with the url and port that the simulator output before, as shown in the image. The URL may differ from pictured. Then, click `Open` in the bottom right corner. The left sidebar should show that a connection has been made. The map should update and the car pose arrow should appear in the map panel in the center. The car can be driven using the right teleop panel.
```

Comment:
the arrow keys/wsad are not working, am I using an old version of foxglove?

Reply:
I left a feature request [for WASD Key Driving](https://github.com/foxglove/studio/issues/2747) for Foxglove Studio, but it looks like it hasn't been implemented yet.